src/_includes/layouts/base.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- site-wide stylesheets -->
+    <link rel="stylesheet" href="/css/reset.css">
+    <link rel="stylesheet" href="/css/theme.css">
+    <link rel="stylesheet" href="/css/helpers.css">   
+    <link rel="stylesheet" href="/css/layout.css">
+    <link rel="stylesheet" href="/css/base.css">
+    <!-- contains core js functionality for all examples -->
+    <script src="/js/main.js" defer></script>
+    <!-- styles for syntax highlighting  -->
+    <link
+    href="https://unpkg.com/prismjs@1.20.0/themes/prism-okaidia.css"
+    rel="stylesheet"
+    media="(prefers-color-scheme: dark)"
+    />
+    <link
+    href="https://unpkg.com/prismjs@1.20.0/themes/prism.css"
+    rel="stylesheet"
+    media="(prefers-color-scheme: light)"
+    />
+  <!-- allow extra tags in the head -->
+  {% if extraHead %}
+    {% for tag in extraHead %}
+      {{ tag }}
+    {% endfor %}
+  {% endif %}
+  <title>{{ title }}</title>
+</head>
+<body>
+   <div class="container">
+      {% include "partials/nav.html" %}
+      
```

Comment:
is this extra space required because partials/nav.html would be included above and break the formatting?